README.md

```diff
@@ -59,10 +59,20 @@ Options:
                                          (1 = DEBUG, 2 = INFO, 3 = WARN)
        -v                              print the version number and exit
 
-Examples:
+### Examples:
 
-     $ nrsc5 -p 63 -g 490 -w samples1071 107.1 0
```

Comment:
I removed the `-p` option from this example because there's no need to manually set PPM correction now that nrsc5 corrects the frequency offset automatically.